From aa615948c11dd9e758574ac1f4c8e523a34a09c6 Mon Sep 17 00:00:00 2001
From: Harry Youd <harry@harryyoud.co.uk>
Date: Mon, 5 Mar 2018 22:36:33 +0100
Subject: [PATCH 02/20] [DNM] updater: Don't check fingerprint for incrementals

TWRP doesn't have the same properties as the underlying system

Instead of mounting /system and parsing build.prop, just screw it and let
the block verification do it instead

Only apply to v3+ blockimgdiff because if we do this on <=2, as move
operations aren't checked

Change-Id: I3114c7bb3b6dc26b4ecd85b726a004b94521db67
---
 tools/releasetools/ota_from_target_files.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tools/releasetools/ota_from_target_files.py b/tools/releasetools/ota_from_target_files.py
index e6931c3f2..de92cdda9 100755
--- a/tools/releasetools/ota_from_target_files.py
+++ b/tools/releasetools/ota_from_target_files.py
@@ -803,7 +803,7 @@ else if get_stage("%(bcb_dev)s") != "3/3" then
 
   else: # blockimgdiff_version > 2
     if source_oem_props is None and target_oem_props is None:
-      script.AssertSomeFingerprint(source_fp, target_fp)
+      pass
     elif source_oem_props is not None and target_oem_props is not None:
       script.AssertSomeThumbprint(
           GetBuildProp("ro.build.thumbprint", OPTIONS.target_info_dict),
-- 
2.17.1

